src/codegen/tests/function_tests.rs

```diff
@@ -403,3 +403,38 @@ fn argument_fed_by_ref_then_by_val() {
 
     insta::assert_snapshot!(result)
 }
+
+#[test]
+fn properties_are_not_registered_as_variables_in_a_pou() {
```

Comment:
This needs fixing, as the codegen function doesn't make any use of lowering therefore always returning a correct result here.